README.md

```diff
@@ -39,6 +39,9 @@ writing some simple queries.
 ### Requirements
 
 - Node.js 18+
+  - The driver requires the Browser API `crypto` to be available as a global.
+    Therefore, for Node 18, you _must_ run `node` with the special command line
```

Comment:
Since you mention 18+ above, I would rephrase this to clarify that _only_ 18 needs the flag, as the feature stabilized in 20+

Reply:
Instead of "Therefore, for Node 18," maybe "Therefore, for Node 18 only,"?

Reply:
Or maybe "Therefore, for Node versions before 19,"?